mkdocs.yml

```diff
@@ -2,11 +2,12 @@ site_name: OSG Production
 site_url: https://opensciencegrid.org/production/
 repo_url: https://github.com/opensciencegrid/production/
 theme: 
-  name: readthedocs
+  name: material
```

Comment:
Could you get rid of this theme block? It's redundant with the one below (I think I borked this last time I updated it).